k8s/model-crd.yaml

```diff
@@ -3,6 +3,11 @@ kind: CustomResourceDefinition
 metadata:
   name: models.equinor.com
 spec:
+  additionalPrinterColumns:
```

Comment:
Remember to add a similar thing for gordo-crd as well showing the nr of defined models when that is added.